.travis.yml

```diff
@@ -31,8 +31,9 @@ matrix:
       env:
         - DEPENDS="cython numpy scipy matplotlib h5py nibabel cvxopt scikit_learn"
 before_install:
+    - apt-get update && apt-get install -y python-vtk
```

Comment:
It's a good idea to do that, but maybe using sudo for one build would be enough for now.